README.md

```diff
@@ -1,11 +1,10 @@
 # Pragma
 
-An official theme for OJS 3.2+.
-
+An official theme for [OJS 3.6](https://pkp.sfu.ca/ojs/).
```

Comment:
Since this is the landing page for the plugin on GitHub, I think it should be clear the theme works for previous versions to avoid confusion from new users.